packages/hathor-rpc-handler/src/rpcMethods/createNanoContractCreateTokenTx.ts

```diff
@@ -81,6 +81,7 @@ export async function createNanoContractCreateTokenTx(
     actions: data?.actions ?? [],
     method,
     args: data?.args ?? [],
+    parsedArgs: [],
```

Comment:
This is just so this matches the new type. I will change this rpc method in another PR